com.unity.render-pipelines.universal/Documentation~/rendering/deferred-rendering-path.md

```diff
@@ -0,0 +1,422 @@
+
+Turning the option on does not increase the memory footprint. To store the normals, Unity uses the same RGB channel in the normal texture regardless of the encoding.
+
+### G-buffer layout
+
+This section describes how Unity stores material attributes in the G-buffer in the Deferred Rendering Path.
+
+The following illustration shows the data structure for each pixel of the render targets that Unity uses in the Deferred Rendering Path.
+
+![Data structure of the render targets that Unity uses in the Deferred Rendering Path](../Images/rendering-deferred/data-structure-render-targets-g-buffer.png)
+
+The data structure consists of the following components.
+
+**Albedo (sRGB)**
+
+This field contains the albedo color in sRGB format, 24 bits.
+
+**MaterialFlags**
+
+This field is a bit field that contains Material flags:
+
+* Bit 1, **ReceiveShadowsOff**: if set, the pixel does not receive dynamic shadows.
+
+* Bit 2, **SpecularHighlightsOff**: if set, the pixel does not receive specular highlights.
+
+* Bit 4, **SubtractiveMixedLighting**: if set, the pixel uses subtractive mixed lighting.
+
+* Bit 8, **SpecularSetup**: if set, the Material uses the specular workflow.
+
+For more technical details, see the file `/ShaderLibrary/UnityGBuffer.hlsl`.
+
+**Specular**
+
+This field contains the following values:
+
+* SimpleLit Material: RGB specular color stored in 24 bits.
+
+* Lit Material with metallic workflow: reflectivity stored in 8 bits, 16 bits are not used.
+
+* Lit Material with specular workflow: RGB specular color stored in 24 bits.
+
+**Occlusion**
+
+This field contains the baked occlusion value from the baked lighting. For real-time lighting, Unity calculates the ambient occlusion value by combining the baked occlusion value with the SSAO value.
+
+**Normal**
+
+This field contains the world space normals encoded in 24 bits. For information on the encoding of normals, see section [Encoding of normals in G-buffer](#accurate-g-buffer-normals).
+
+**Smoothness**
+
+This field stores the smoothness value for the SimpleLit and Lit materials.
+
+**Emissive/GI/Lighting**
+
+This render target contains the Material emissive output and baked lighting. Unity fills this field during the G-buffer Pass. During the deferred shading pass, Unity stores lighting results in this render target.
+
+Render target format:
+
+* **B10G11R11_UFloatPack32**, unless one of the following conditions is true:
+
+    * In URP Asset, the setting **Quality** > **HDR** is turned on, and the target Player platform does not support HDR.
+
+    * In Player Settings, the setting **PreserveFramebufferAlpha** is true.
+
+* **R16G6B16A16_SFloat**, if Unity cannot use **B10G11R11_UFloatPack32** because of the project settings.
+
+* If Unity cannot use one of the other formats in the list, it uses what the following method returns: `SystemInfo.GetGraphicsFormat(DefaultFormat.HDR)`.
```

Comment:
In LDR mode, Emissive/GI/lighting render-target will be the same as forward renderer. It seems to be `GraphicsFormat.R8G8B8A8_SRGB`.

@oleks-k  I suppose we should document the formats of each render-target:

**Albedo(sRGB), MaterialFlags:** `GraphicsFormat.R8G8B8A8_SRGB` or `GraphicsFormat.R8G8B8A8_UNorm` (depends on theQualitySettings color space)

**Specular, Occlusion:** `GraphicsFormat.R8G8B8A8_UNorm`

**Normal, Smoothness:** `GraphicsFormat.R8G8B8A8_UNorm` or `GraphicsFormat.R8G8B8A8_SNorm`: depends on accurateNormals setting (`GraphicsFormat.R8G8B8A8_SNorm` is preferred because terrain layers blend with less artefacts).

**Emissive/GI/Lighting:** already explained (in essence camera color target)

**ShadowMask:** `GraphicsFormat.R8G8B8A8_UNorm`

**Rendering Layer Mask:** `GraphicsFormat.R8_UNorm`. It may change in the future when more bits are required to store layer bits.